File: python/1 - Iniciante/1065_pares_entre_cinco_numeros.py
A = int(input())
B = int(input())
C = int(input())
D = int(input())
E = int(input())

contador = 0
N = [A, B, C, D, E]
for num in N:
    if num % 2 == 0:
        contador += 1

print(f'{contador} valores pares')

File: c/c99/1 - Iniciante/1065_pares_entre_cinco_numeros.c
#include <stdio.h>

int main() {
    int N, pares = 0;
    for (int i = 0; i < 5; i++) {
        scanf("%d", &N);
        if (N % 2 == 0) pares++;
    }

    printf("%d valores pares\n", pares);
    return 0;
}
